specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1438,17 +1438,206 @@
                 }
             }
         },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/northSouthHardenings": {

```

Comment:
For these GET apis, I would assume that the response could be quite large, as each instance includes a number of rules and a number of alerts. It may be prudent to return only the most relevant data on GET, unless a querystring parameter ($expand) is passed. Additionally, supporting a $filter property (described [here](https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#97-filtering)) can be helpful to reduce the response size.